intents_consolidator/src/intents/consolidator/handlers/facial_recognition_handler.cpp

```diff
@@ -15,16 +16,32 @@ namespace consolidator
 {
 
 FacialRecognitionResult::FacialRecognitionResult(
-  const machine_intents_interfaces::msg::FacialRecognitionResult * msg)
+  const machine_intents_interfaces::msg::FacialRecognitionResult * msg, const entity::Pose pose)
 {
   embedding_ = std::make_unique<datastructures::Embedding>(msg->embedding);
+  pose_ = pose
 }
 
 
 void FacialRecognitionHandler::Merge(FacialRecognitionResult & res, State & state)
 {
-  //
-  return false;
+  double acceptable_error = 0.02;
+  bool matched = false;
+  for(auto & element : state.Occupants()) {
+    double comparison = res.embedding_.Compare(element.GetEmbedding());
+    if(abs(comparison) < acceptable_error) {
```

Comment:
Is abs the right thing to use here? I believe we should be using std::abs for C++. Fact check me here @rithvikp